src/main.c

```diff
@@ -1,48 +1,264 @@
 /*
- * Copyright (c) 2016 Intel Corporation
- *
+ * Copyright (c) 2024, CATIE
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include <inttypes.h>
+#include <lvgl.h>
 #include <stdio.h>
-#include <zephyr/kernel.h>
+#include <string.h>
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/display.h>
 #include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/pwm.h>
+#include <zephyr/input/input.h>
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/sys/printk.h>
+#include <math.h>
```

Comment:
All these includes are necessary? For example, I think `<zephyr/sys/printk.h>` is useless 🤔